### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
## What do these changes do?

Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/GraphScope/actions/runs/7936794632/attempts/1#summary-21672807577

The action will report that the changes in this PR would make it happy: https://github.com/jsoref/GraphScope/actions/runs/7936795177/attempts/1#summary-21672809066

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ If applicable, add screenshots to help explain your problem.
 
 **Environment (please complete the following information):**
  - GraphScope version: [e.g., v0.1, v0.2.2, master]
- - OS: [e.g. MacOS, Linux]
+ - OS: [e.g. macOS, Linux]
```
Comment (jsoref) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: brand

File: .github/workflows/gaia.yml [hunk 2]
```diff
@@ -88,12 +88,12 @@ jobs:
       run: |
         . ${HOME}/.graphscope_env
         git clone -b master --single-branch --depth=1 https://github.com/7br/gstest.git /tmp/gstest
-        cd ${GITHUB_WORKSPACE}/interactive_engine/compiler && ./ir_exprimental_ci.sh
+        cd ${GITHUB_WORKSPACE}/interactive_engine/compiler && ./ir_experimental_ci.sh
```
Comment (jsoref) [inline .github/workflows/gaia.yml:91]: I've tried to rename related files...

File: .github/free-disk-space.sh [hunk 3]
```diff
@@ -22,7 +22,7 @@
 # Free: 17 GB
 # This script frees up 28 GB of disk space by deleting unneeded packages and 
 # large directories.
-# The Flink end to end tests download and generate more than 17 GB of files,
+# The following end to end tests download and generate more than 17 GB of files,
```
Comment (jsoref) [inline .github/free-disk-space.sh:25]: It took me a while to decide that this was probably the intended word

**Review by jsoref — COMMENTED** [review]
I'll try to add notes about the rest of the changes later... It'll take me a while.

Note: I'm more than happy to split things off / exclude items. I just need input.

**Comment by sighingnow** [comment]
Closing as we cannot accept such a monster PR for spelling.

**Comment by jsoref** [comment]
I'm happy to split it up in any manner you request

**Comment by sighingnow** [comment]
> I'm happy to split it up in any manner you request

We currently only consider typos in documentation and comments. The typo in variables in sources and YAML/JSON configurations requires further case-by-case investigation.

Thanks!

**Comment by jsoref** [comment]
I've split off most of the comments changes. If that manages to be merged, I'll see about making a PR for doc changes, and we can go from there.

Note that it's purely a heuristic, so there may be some comment fixes that I dropped.

**Comment by sighingnow** [comment]
> ## What do these changes do?
> Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).
> 
> The misspellings have been reported at https://github.com/jsoref/GraphScope/actions/runs/7936794632/attempts/1#summary-21672807577
> 
> The action will report that the changes in this PR would make it happy: https://github.com/jsoref/GraphScope/actions/runs/7936795177/attempts/1#summary-21672809066
> 
> ## Related issue number

Hi @jsoref, thanks a lot for those spelling fixes. From the link you pasted in the description, I cannot only find the list of unrecognized words. I'm curious about how the commits (applying suggestions) are generated?

I would like to add your check-spelling action to this repo as well as our other repos as well. Could you please show me some wiki/docs about it?

Thanks!

**Comment by sighingnow** [comment]
> Depending on the project and my current work-mode, I take different approaches to correcting misspellings. Sometimes I'll manually correct things, sometimes I let chrome suggest a correction, and sometimes I let Google Sheets suggest corrections.

I see. Thanks again for you effort!

**Pull request closed without merging**